corehq/apps/export/migrations/0014_deidmapping.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 4.2.17 on 2025-02-12 00:46
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("export", "0013_rm_incrementalexport"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="DeIdMapping",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
```

Comment:
How many of these do we expect to have? If the number grows very large, especially if there is a lot of churn, would it be worth using `BigAutoField`?

Reply:
I don't have a sense of how widely the de-identified data feature is used, but it seems plausible we could run up against the limit using if a regular `AutoField`. If we consider exports from our larger projects that might have several hundred thousand unique cases, if each of those has a few different de-identified fields, and a few different projects of that scale use this feature, it would add up pretty quickly, so `BigAutoField` is likely worthwhile.

Reply:
I feel like this suggestion of using `BigAutoField` may have pointed in a direction I'm not really in favor of. Thinking about this more, I'm strongly in favor of using a different hashing technique that doesn't require a new database model, such as using a unique salt per domain. Yet another idea is to use the document ID as hash salt.